Assets/Scripts/Bullets/BulletController.cs

```diff
@@ -1,24 +1,39 @@
-﻿using UnityEngine;
+﻿using System.Collections;
+using UnityEngine;
 
 public class BulletController : MonoBehaviour
 {
+    [SerializeField]
+    private ParticleSystem explosionParticle;
+    [SerializeField]
+    private AudioSource bulletExplosionAudio;
```

Comment:
There should generally be one audio source and you should use that to play different audio clips